.drone.yml

```
@@ -593,8 +593,8 @@ steps:
 services:
 - name: cache
   image: redis
-- name: postgres-9
-  image: postgres:9
+- name: postgres-9.6
```

```suggestion
- name: postgres-9
```


in `autotest.sh` it waits for a host named `postgres-$POSTGRES`